components/suggest/src/db.rs

```diff
@@ -182,6 +182,7 @@ impl<'a> SuggestDao<'a> {
                 let title = row.get("title")?;
                 let raw_url = row.get::<_, String>("url")?;
                 let provider = row.get("provider")?;
+                let score = row.get::<_,Option<f64>>("score")?;
```

Comment:
```suggestion
                let score = row.get::<_, f64>("score")?;
```

Oops, sorry I didn't catch this! Since you're using `IFNULL` in the query, this doesn't need to be an `Option` anymore—it'll never be `None`.